.circleci/choose-e2e-env.sh

```diff
@@ -11,7 +11,7 @@ if [[ ! -z $CIRCLE_PULL_REQUEST ]]; then
   # If e2e files are the only files that have changed, update WORKBENCH_ENV to "test"
   if [ $E2E_CHANGED_COUNT -gt 0 ] && [ $E2E_CHANGED_COUNT -eq $CHANGED_COUNT ]; then
```

Comment:
FYI: not for this PR, but while we're here my other request per https://precisionmedicineinitiative.atlassian.net/browse/RW-6871 is to eliminate this "e2e only" distinction.

Reply:
Good points. I'll make changes gradually. Thank you.